java/piranha/src/test/java/com/uber/piranha/ConfigurationTest.java

```diff
@@ -1626,6 +1628,106 @@ public void exceptionWhenArgumentIndexNotSpecifiedAndNotArgumentIndexOptional()
     flagCleaner.init(b.build());
   }
 
+  /**
+   * Helper method for "enumProperties" related tests. All config tests for this expect the code to
+   * be unchanged.
+   *
+   * <p>The test to check the "enumProperties" refactoring functionality is in {@link
+   * CorePiranhaTest#testRemoveEnumFieldValueMatch()}
+   */
+  private BugCheckerRefactoringTestHelper enumUnchangedTest(BugCheckerRefactoringTestHelper bcr) {
+    return bcr.addInputLines(
+            "XPFlagCleanerRemoveMatchingEnum.java",
+            "package com.uber.piranha;",
+            "import org.junit.Test;",
+            "import static com.uber.piranha.TestExperimentName.STALE_FLAG;",
+            "class XPFlagCleanerRemoveMatchingEnum {",
+            "  private XPTest experimentation;",
+            "  @Test",
+            "  public void test_StaleFlag_treated() {",
+            "     experimentation.putToggleEnabled(STALE_FLAG.getKey());",
+            "  }",
+            "}")
+        .expectUnchanged();
+  }
+
+  /**
+   * [https://github.com/uber/piranha/issues/141]
```

Comment:
One line description here? (I like the link, but it should be possible to get some idea what this is about without following the link. Also, if we ever move away from GitHub for issue tracking or relocate to another repo/org, some documentation should remain in the code itself)

Reply:
Actually, looking at the paragraph below, this is fine, I think... though maybe I would still do for these tests:

```
Testing enumProperties configuration.

<p>[the test specific paragraph you have written for each test]

<p> See also: [https://github.com/uber/piranha/issues/141]
```